tests/unit_tests/test_wave_06.py

```diff
@@ -143,16 +148,22 @@ def test_swap_best_by_category_reordered():
         their_priority="Decor"
     )
 
-    raise Exception("Complete this test according to comments below.")
+    #raise Exception("Complete this test according to comments below.")
     # *********************************************************************
     # ****** Complete Assert Portion of this test **********
     # *********************************************************************
     # Assertions should check:
     # - That result is truthy
+    assert result is True
     # - That tai and jesse's inventories are the correct length
+    assert len(tai.inventory) == 3
+    assert len(jesse.inventory) == 3
     # - That all the correct items are in tai and jesse's inventories, and that the items that were swapped are not there
+    assert item_f in tai.inventory
+    assert item_c in jesse.inventory
```

Comment:
We could use a for loop to check to see if all the items are in their respective inventories.